libs/model/src/contest/UpdateContestManagerMetadata.command.ts

```diff
@@ -0,0 +1,25 @@
+import type { Command } from '@hicommonwealth/core';
+import { schemas } from '@hicommonwealth/core';
+import { models } from '../database';
+import { mustExist } from '../middleware/guards';
+
+export const UpdateContestManagerMetadata: Command<
+  typeof schemas.commands.UpdateContestManagerMetadata
+> = () => ({
+  ...schemas.commands.UpdateContestManagerMetadata,
+  auth: [],
```

Comment:
same

Reply:
@rbennettcw make sure auth requirements are updated. Let's discuss how to properly gather "ticket" requirements before implementation/PR reviews.